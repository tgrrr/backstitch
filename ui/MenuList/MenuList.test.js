import React from 'react'
import renderer from 'react-test-renderer'
import { MenuList } from '.'
import { MenuItem } from '../index'

it('MenuList: default', () => {
    const component = renderer.create(
        <MenuList>
            <MenuItem>Foo</MenuItem>
            <MenuItem>Bar</MenuItem>
        </MenuList>
    )
  const tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
