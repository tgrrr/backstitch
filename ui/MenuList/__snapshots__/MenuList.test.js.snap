// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MenuList: default 1`] = `
<div
  className="sc-AxjAm StDqN"
>
  <div
    className="Menulist "
    data-testid="Menulist"
  >
    <ul
      className="MuiList-root MuiList-padding"
      onKeyDown={[Function]}
      role="menu"
      tabIndex={-1}
    >
      MenuList
    </ul>
  </div>
</div>
`;
